README.md

```diff
@@ -16,8 +16,9 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
-* >= 8.1
+##### Minimum ownCloud / Nextcloud Version
```

Comment:
The code of ownCloud and Nextcloud hasn't diverged in any way, that affects the notes app, yet.
So I would like to still keep this information in the README.
But we might drop support for ownCloud in the future, if something significant changes. We can always say "ownCloud version 8.1 - ??.??". But for now it works and I'm happy to support it.